aip/0131.md

```diff
@@ -14,9 +14,9 @@ In REST APIs, it is customary to make a `GET` request to a resource's URI (for
 example, `/v1/shelves/{shelf}/books/{book}`) in order to retrieve that
 resource.
 
-Resource-oriented design ([AIP-121][]) honors this pattern through the
-`Get` method. These RPCs accept the URI representing that resource and return
-the resource.
+Resource-oriented design ([AIP-121][]) honors this pattern through the `Get`
```

Comment:
This is an unrelated formatting change from prettier (I have "format on save" turned on).

Reply:
OK -- thanks for the comment here.